src/jsMain/kotlin/io/beatmaps/maps/mapinfo.kt

```diff
@@ -2,11 +2,7 @@ package io.beatmaps.maps
 
 import external.Axios
 import external.generateConfig
-import io.beatmaps.api.CurateMap
-import io.beatmaps.api.MapDetail
-import io.beatmaps.api.MapInfoUpdate
-import io.beatmaps.api.SimpleMapInfoUpdate
-import io.beatmaps.api.StateUpdate
+import io.beatmaps.api.*
```

Comment:
Please tell intelliJ to not

Reply:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports
(But for Kotlin instead of Java)